### PR TITLE
Feature/template update

### PR DESCRIPTION
**Pull request by jobenjada**
## What does this PR do?

- Updated templates and added new ones.
- Revamped the Template view (kicked out Icons, added colored tags)

grafik

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change adds a new database migration
- [ ] This change requires a documentation update

## How should this be tested?

- Create a survey with a template

## Checklist

- [X] Read the [contributing guide](https://github.com/formbricks/formbricks/blob/main/CONTRIBUTING.md)
- [X] Self-reviewed my own code
- [X] Commented on my code in hard-to-understand bits
- [X] Ran `pnpm build`
- [X] Checked for warnings, there are none
- [X] Removed all `console.logs`
- [X] Merged the latest changes from main onto my branch with `git pull origin main`
- [X] Updated the Formbricks Docs if changes were necessary

### CODE REVIEW

File: apps/web/app/s/[surveyId]/LinkSurvey.tsx [hunk 1]
```diff
@@ -72,13 +72,13 @@ export default function LinkSurvey({ survey }: LinkSurveyProps) {
       case "notEquals":
         return answerValue !== logic.value;
       case "lessThan":
-        return answerValue < logic.value;
+        return logic.value !== undefined && answerValue < logic.value;
```
Comment (mattinannt) [inline apps/web/app/s/[surveyId]/LinkSurvey.tsx:75]: Do this changes also need to be made in `packages/js/src/components/SurveyView.tsx`?

**Review by mattinannt — CHANGES_REQUESTED** [review]
Left a comment in the code.

I really like the simplicity and branch icon in the template view :-) 

I also noticed some contrast problems due to the new color theme. More contrast would be nice :-) 
Screenshot 2023-06-07 at 17 45 10

There seems to be a small issue with padding when tags are across multiple lines:

Screenshot 2023-06-07 at 17 50 04

File: apps/web/app/s/[surveyId]/LinkSurvey.tsx (hunk 1 above)
Comment (jobenjada) [reply apps/web/app/s/[surveyId]/LinkSurvey.tsx:75]: @moritzrengert I think you made this change, no? Please clarify 🙌

Comment (jobenjada) [reply apps/web/app/s/[surveyId]/LinkSurvey.tsx:75]: @moritzrengert I think you made this change. I have updated the SurveyView and tested the lessThan condition but please confirm that its correct. And why is the check for undefined needed for lessThan? And why not for the others? 🤓

Comment (jobenjada) [reply apps/web/app/s/[surveyId]/LinkSurvey.tsx:75]: got it, thanks :)

**Review by mattinannt — APPROVED** [review]
great job, looks good 🤗🎉

**Pull request merged**